### PR TITLE
feat(ru-KZ): autogenerated locale ru-KZ. Closes #108

### LINKED ISSUE #108: Locale request: ru-KZ
**Which locale do you wish to add?**
First check if your locale has already been requested. If so, go and add an emoji to that request to show your support.

For new requests, write the locale identifier in the subject after the colon.

Example: "**Locale request: nb-NO**".

We use BCP-47 with region subtags, [see "IETF language tag"](https://en.wikipedia.org/wiki/IETF_language_tag).

**Maintainer**
Sanity will bootstrap and keep the locale updated using AI. We would like human contributors to help maintain the locales by improving them and/or reviewing pull requests from the AI and from other contributors as they come in. Check the box below if you want to be added as a maintainer of this locale. As a maintainer you will be automatically added as reviewer to future PRs that improve it.

If you are not the author of this issue, but would still like to be added as a maintainer, please add a comment to the thread.

- [x] I want to be added as maintainer

### PR DESCRIPTION
**Pull request by runeb**
Proposes initial auto translation for `ru-KZ` with @yenbekbay as maintainer

### CODE REVIEW

File: locales/ru-KZ/src/studio.ts [hunk 1]
```diff
@@ -0,0 +1,1528 @@
...
+    'Этот файл не может быть удален, так как он используется. Чтобы удалить его, сначала нужно удалить все его использования.',
+  /** Warning message showing when *named* image can't be deleted because it is in use */
+  'asset-source.delete-dialog.usage-list.warning-image-is-in-use_named':
+    '{{filename}} не может быть удалено, так как оно используется. Чтобы удалить это изображение, сначала нужно удалить все его использования.',
+  /** Warning message showing when *unnamed* image can't be deleted because it is in use */
+  'asset-source.delete-dialog.usage-list.warning-image-is-in-use_unnamed':
+    'Это изображение не может быть удалено, так как оно используется. Чтобы удалить его, сначала нужно удалить все его использования.',
+  /** Text shown when the list of assets only include a specific set of types */
+  'asset-source.dialog.accept-message':
+    'Показаны только активы принятых типов: <strong>{{acceptTypes}}</strong>',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/studio.ts:52]:
```suggestion
    'Показаны только ресурсы принятых типов: <strong>{{acceptTypes}}</strong>',
```

File: locales/ru-KZ/src/studio.ts [hunk 2]
```diff
@@ -0,0 +1,1528 @@
...
+   * eg "Search for all types".
+   */
+  'search.action.search-all-types': 'Искать все документы',
+  /**
+   * Text displayed when we are able to determine one or more document types that will be used for
+   * searching, and can fit within the space assigned by the design.
+   */
+  'search.action.search-specific-types': 'Искать {{types, list}}',
+  /** Dialog title for action to select an asset of unknown type */
+  'search.action.select-asset': 'Выбрать актив',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/studio.ts:1024]:
```suggestion
  'search.action.select-asset': 'Выбрать ресурс',
```

File: locales/ru-KZ/src/studio.ts [hunk 3]
```diff
@@ -0,0 +1,1528 @@
...
+  'search.error.display-filter-description':
+    'Это может указывать на неверные опции, определенные в вашей схеме.',
+  /** Title for error when a filter cannot be displayed (mainly a developer-oriented error) */
+  'search.error.display-filter-title': 'Произошла ошибка при отображении этого фильтра.',
+  /** Description for error when no valid asset source is found, describes that you should check the the current studio config */
+  'search.error.no-valid-asset-source-check-config-description':
+    'Пожалуйста, убедитесь, что он включен в файле конфигурации вашей студии.',
+  /** Description for error when no valid asset source is found, describes that only the default asset is supported */
+  'search.error.no-valid-asset-source-only-default-description':
+    'В настоящее время поддерживается только источник активов по умолчанию.',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/studio.ts:1056]:
```suggestion
    'В настоящее время поддерживается только источник ресурсов по умолчанию.',
```

File: locales/ru-KZ/src/studio.ts [hunk 4]
```diff
@@ -0,0 +1,1528 @@
...
+  /** Title for error when no valid asset sources found */
+  'search.error.no-valid-asset-source-title': 'Не найдены действительные источники активов.',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/studio.ts:1058]:
```suggestion
  'search.error.no-valid-asset-source-title': 'Не найдены действительные источники ресурсов.',
```

File: locales/ru-KZ/src/validation.ts [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  'number.maximum': 'Должно быть меньше или равно {{maxNumber}}',
+  /** Number has more precision (decimals) than the allowed "$limit" */
+  'number.maximum-precision': 'Максимальная точность {{limit}}',
+  /** Number is lower than the given minimum value "$minNumber" */
+  'number.minimum': 'Должно быть больше или равно {{minNumber}}',
+  /** Number is not an integer ("whole number") */
+  'number.non-integer': 'Должно быть целым числом',
+
+  /** Object is missing a reference to an asset document in its `asset` field */
+  'object.asset-required': 'Требуется актив',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/validation.ts:49]:
```suggestion
  'object.asset-required': 'Требуется ресурс',
```

File: locales/ru-KZ/src/vision.ts [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  'query.error.column': 'Столбец',
+  /** Label for 'Line' indicator when there is an error within the query */
+  'query.error.line': 'Строка',
+  /** Label for "Query" editor/input */
+  'query.label': 'Запрос',
+  /** Label for the "Query URL" field, shown after executing a query, and allows for copying */
+  'query.url': 'URL запроса',
+
+  /** Label for "End to End time" information of the fetched query */
+  'result.end-to-end-time-label': 'Конец-к-концу',
```
Comment (yenbekbay) [inline locales/ru-KZ/src/vision.ts:36]:
```suggestion
  'result.end-to-end-time-label': 'Время от начала до конца',
```

**Review by yenbekbay — CHANGES_REQUESTED** [review]
Everything looks good — just a few suggestions.

**Comment by runeb** [comment]
Thank you for your review, @yenbekbay!

**Pull request merged**